game/ato/flightroster.py

```diff
@@ -22,6 +22,10 @@ def squadron(self) -> Squadron:
     def iter_pilots(self) -> Iterator[Pilot | None]:
         yield from self.pilots
 
+    @property
+    def player_count(self) -> int:
```

Comment:
My preference would be to add a `has_clients` method which will use the same statement as you currently have here, expect for using `any` instead of `len`, although I doubt that the performance gain will be worth it so I'll leave it up to you to decide on that...

Reply:
This method is an original one from Liberation, which has been refactored out at some point, I simply reintroduced it. This leads me to believe that it migh have had some use previously and I would prefer to keep it as is, since it might become useful again further down the line. Also, as you said, the performance gain would not be significant.